src/Disks/IDiskRemote.cpp

```diff
@@ -102,6 +143,17 @@ IDiskRemote::Metadata::Metadata(
     }
 }
 
+/// Load metadata by path or create empty if `create` flag is set.
```

Comment:
Yes, what do you thing about using local fs hardlink count? Why do you think it is bad idea?

Reply:
I don't fully understand the purpose of userland hardlink tracking. However, if we somehow need to do it properly, we will have to synchronize every file operation which can change the `hardlink` property. If the mutex is per-disk, it will be too heavy, since some common mutation like DROP COLUMN can generate tons of hardlinks. I don't know how good `flock` is, but it smells bad.